README.md

```diff
@@ -59,6 +74,10 @@ By default the DIVA reference implementation runs with `in memory` session manag
 - `REDIS_PORT`: the port where the redis instance can be reached
 - `REDIS_PASSWORD`: the password as configured in redis
 
+### Bing maps API key
+
+The MyHome example in the [diva-js-reference-3p-frontend](https://github.com/Alliander/diva-js-reference-3p-frontend) frontend displays a map image that uses the Bing maps API through this backend. To make it work, provide your own private bing maps api key as an export, or set a placeholder (of course then the Bing API won't work correctly).
```

Comment:
Maybe add a link to Bing on how to get such an API key https://msdn.microsoft.com/en-us/library/ff428642.aspx